src/transformers/generation/candidate_generator.py

```diff
@@ -732,15 +735,17 @@ def get_candidates(self, input_ids: torch.LongTensor) -> Tuple[torch.LongTensor,
         generation_args["generation_config"].return_dict_in_generate = True
 
         # Generate and process outputs using translator
+        generation_args['logits_processor'] = self._atm_translator.logits_processors
         assistant_output = self.assistant_model.generate(**generation_args, **self.assistant_kwargs)
         self.assistant_kwargs["past_key_values"] = assistant_output.past_key_values
 
         candidate_logits = torch.stack(assistant_output.scores, dim=1)
 
         # Use translator to convert tokens and logits
         candidate_ids = assistant_output.sequences
-        candidate_logits = self._atm_translator.logits_processors(input_ids=candidate_ids, scores=candidate_logits)
+        self._prev_assistant_ids = candidate_ids
         target_ids = self._atm_translator.get_target_ids(assistant_input_ids, target_input_ids, candidate_ids)
+        self._candidates_target_seq_len = target_ids.shape[-1]
```

Comment:
Here, `self._candidates_target_seq_len` is set to the length of all the given sequence rather than the length of only the candidates, right? If so, the name `self._candidates_target_seq_len` might not be most informative, and even confusing. Please consider using the original name (`self._prev_target_seq_len`) or propose a better alternative.

Reply:
`self._candidates_target_seq_len` is that length of target seq length including draft candidates before validation. 'self._prev_target_seq_len' length of target seq length including validated tokens from previous spec iteration. Renamed `self._candidates_target_seq_len` to `self._target_seq_len_with_candidates`